src/Stratis.Bitcoin.Features.Consensus/ConsensusController.cs

```diff
@@ -83,13 +73,13 @@ public IActionResult DeploymentFlags()
         /// </summary>
         /// <returns>Json formatted <see cref="uint256"/> hash of the block at the consensus tip. Returns <see cref="IActionResult"/> formatted error if fails.</returns>
         /// <remarks>This is an API implementation of an RPC call.</remarks>
-        [Route("api/[controller]/getbestblockhash")]
+        [Route("getbestblockhash")]
         [HttpGet]
-        public IActionResult GetBestBlockHashAPI()
+        public IActionResult GetBestBlockHash()
         {
             try
             {
-                return this.Json(this.GetBestBlockHashRPC());
+                return this.Json(this.ChainState.ConsensusTip?.HashBlock);
```

Comment:
seems to me this breaks the previous return type no?

Reply:
This is a result of the ConsensusController split to separate API & RPC controller's as the attribute for the API controllers that are required in the new version of swagger break the RPC.

Reply:
ah gotcha ok fine